QuantLib/ql/termstructures/volatility/volatilitynature.hpp

```diff
@@ -0,0 +1,49 @@
+/* -*- mode: c++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+
+/*
+ Copyright (C) 2015 Peter Caspers
+
+ This file is part of QuantLib, a free-software/open-source library
+ for financial quantitative analysts and developers - http://quantlib.org/
+
+ QuantLib is free software: you can redistribute it and/or modify it
+ under the terms of the QuantLib license.  You should have received a
+ copy of the license along with this program; if not, please email
+ <quantlib-dev@lists.sf.net>. The license is also available online at
+ <http://quantlib.org/license.shtml>.
+
+ This program is distributed in the hope that it will be useful, but WITHOUT
+ ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ FOR A PARTICULAR PURPOSE.  See the license for more details.
+*/
+
+/*! \file volatilitynature.hpp
+    \brief volatility nature descriptions
+*/
+
+#ifndef quantlib_volatility_nature_hpp
+#define quantlib_volatility_nature_hpp
+
+namespace QuantLib {
+
+    enum VolatilityNature { ShiftedLognormal, Normal };
```

Comment:
I would vote for type too, not nature.

On Fri, May 15, 2015 at 4:11 PM, Luigi Ballabio <notifications@github.com>
wrote:

> In QuantLib/ql/termstructures/volatility/volatilitynature.hpp
> <https://github.com/lballabio/quantlib/pull/216#discussion_r30411109>:
>
> > +
> > + This program is distributed in the hope that it will be useful, but WITHOUT
> > + ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
> > + FOR A PARTICULAR PURPOSE.  See the license for more details.
> > +*/
> > +
> > +/*! \file volatilitynature.hpp
> > +    \brief volatility nature descriptions
> > +*/
> > +
> > +#ifndef quantlib_volatility_nature_hpp
> > +#define quantlib_volatility_nature_hpp
> > +
> > +namespace QuantLib {
> > +
> > +    enum VolatilityNature { ShiftedLognormal, Normal };
>
> Oh, I didn't know that [image: :smile:]
> If it's an established name it's ok, then.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lballabio/quantlib/pull/216/files#r30411109>.
>